packages/core/core/src/utils/nuxt/index.ts

```diff
@@ -1,9 +1,12 @@
 import { nuxtContextFactory } from './nuxtContextFactory';
```

Comment:
why context is prefixed with `nuxt` and integration plugin isn't? both are for nuxt

Reply:
you import `integrationPlugin` only in the plugin file of nuxt.js, so that fact actually means that you are using plugin, so It doesn't need to be extra prefixed